include/swift/AST/Decl.h

```diff
@@ -2945,6 +2948,8 @@ class TypeAliasDecl : public GenericTypeDecl {
     return EqualLoc;
   }
 
+  void setEndLoc(SourceLoc e) { EndLoc = e; }
```

Comment:
Where is this called on TypeAliasDecl?

Reply:
Well-spotted! Turns out I didn't bring over that call from ASTScope branch, but I have now. (Failing tests pointed it out.)